notebooks/text_models/solutions/load_text.ipynb

```diff
@@ -0,0 +1,1935 @@
+{
```

Comment:
<ul><li>Some missing "Learn the difference between high"</li><li>"Learn how to preprocess data" -&gt; "Learn how to preprocess text"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Updated. Thank you! </p>